From 480f78e713b9e9a425ad425342afac70420a4043 Mon Sep 17 00:00:00 2001
From: Jeremy Olexa <darkside@gentoo.org>
Date: Wed, 18 Apr 2012 16:30:57 +0000
Subject: [PATCH] Fix warnings from previous patch

Use of uninitialized value $Munin::Common::Defaults::MUNIN_VERSION in
sprintf at /etc/munin/plugins/nginx_status line 110.
---
 plugins/node.d/nginx_request.in |    2 ++
 plugins/node.d/nginx_status.in  |    2 ++
 2 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/plugins/node.d/nginx_request.in b/plugins/node.d/nginx_request.in
index 402ce54..d3933e1 100644
--- a/plugins/node.d/nginx_request.in
+++ b/plugins/node.d/nginx_request.in
@@ -55,6 +55,8 @@ license.  Munin is GPLv2 licensed.
 
 =cut
 
+use Munin::Plugin;
+
 my $ret = undef;
 
 if (! eval "require LWP::UserAgent;"){
diff --git a/plugins/node.d/nginx_status.in b/plugins/node.d/nginx_status.in
index 0d5d966..a427fc0 100644
--- a/plugins/node.d/nginx_status.in
+++ b/plugins/node.d/nginx_status.in
@@ -55,6 +55,8 @@ license.  Munin is GPLv2 licensed.
 
 =cut
 
+use Munin::Plugin;
+
 my $ret = undef;
 
 if (! eval "require LWP::UserAgent;"){
-- 
1.7.3.4

